### DISCUSSION TITLE
Inpaint using Fooocus model possible?

### DISCUSSION THREAD

**Discussion by ZeroCool22** [Panchovix/stable-diffusion-webui-reForge#35] [Q&A] [answered]
I remember some say we could use **Fooocus Inpaint model** in the original Forge, but never tried it.

**1.** Could someone tell if this is possible.
**2.** If yes, what files i need to download?:

![Screenshot_1](https://github.com/user-attachments/assets/6b7577f3-703e-4f2e-ba1a-16438a908fe1)

**3.** I must load the models in Controlnet Inpaint or like a normal model?

Thx.

**Comment by Torcho12** [comment, marked as answer]
I think it is possible. You should download the latest inpaint_v26.fooocus.patch and then put it in the controlnet folder. After that you go into inpaint section on the general tabs in img2img then controlnet. Click the inpaint section in the controlnet and find your model if it is not there click the refresh button to the right to find it manually from the all controlnet models there. However there are inpainting models for sdxl in civitai. Recently I started using the dreamshaper one. You can inpaint decently for 8 - 10 steps.

**Reply by ZeroCool22** [reply]
> I think it is possible. You should download the latest inpaint_v26.fooocus.patch and then put it in the controlnet folder. After that you go into inpaint section on the general tabs in img2img then controlnet. Click the inpaint section in the controlnet and find your model if it is not there click the refresh button to the right to find it manually from the all controlnet models there. However there are inpainting models for sdxl in civitai. Recently I started using the dreamshaper one. You can inpaint decently for 8 - 10 steps.

Thank you.
